lib/portage/manifest.py

```diff
@@ -182,14 +179,19 @@ def __init__(
         self.required_hashes.update(required_hashes)
         self.required_hashes.intersection_update(self.hashes)
 
-        for t in MANIFEST2_IDENTIFIERS:
-            self.fhashdict[t] = {}
+        self.fhashdict = {t: {} for t in MANIFEST2_IDENTIFIERS}
+
         if not from_scratch:
-            self._read()
-        if fetchlist_dict != None:
-            self.fetchlist_dict = fetchlist_dict
-        else:
-            self.fetchlist_dict = {}
+            """Parse Manifest file for this instance"""
```

Comment:
Should this still be a docstring? (not sure)

Reply:
I'll change it.

Reply:
Thanks!